src/pytorch_lightning/tuner/lr_finder.py

```diff
@@ -196,7 +196,7 @@ def lr_find(
     """See :meth:`~pytorch_lightning.tuner.tuning.Tuner.lr_find`"""
     if trainer.fast_dev_run:
         rank_zero_warn("Skipping learning rate finder since fast_dev_run is enabled.")
-        return
+        return  # type: ignore
```

Comment:
> * Line 199: sometimes `mypy` will throw `Return value expected` error when it detects a lone `return` (https://github.com/python/mypy/issues/1003), in which case I added `# type: ignore` to the line.

Thanks for the information! How about returning `None` explicitly following PEP8 instead?

```suggestion
        return None
```